### PR TITLE
Added runekit plugin

### PR DESCRIPTION
**Pull request by ImToxicCode**
This plugin sends http post requests to the runekit api to present data to the user on an application.

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Your plugin entry must have a `warning` defined as it submits data to an external site. Example: https://github.com/runelite/plugin-hub/blob/master/plugins/shared-implings#L3

**Comment by ImToxicCode** [comment]
Sorry about commit b17da0 that was a mistake.

**Comment by ImToxicCode** [comment]
Is there anything else I need to add to the plugin before it can merge? 

Thank you

File: plugins/runekit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+repository=https://github.com/ImToxicCode/runekit.git
+commit=96087ce4e1823a43997741826f69de7d69064f98
+warning=This plugin communicates the stage of crops and birdhouses to a 3rd party server not controlled or verified by the RuneLite Developers.
+disabled=true
```
Comment (raiyni) [inline plugins/runekit:4]: remove

**Review by raiyni — CHANGES_REQUESTED** [review]

**Review by ImToxicCode — COMMENTED** [review]
Removed disabled field

**Comment by raiyni** [comment]
Do not commit that .idea directory.

**Comment by ImToxicCode** [comment]
@raiyni What should I do?

**Comment by Nightfirecat** [comment]
Remove these files from this PR:
* .idea/.gitignore
* .idea/modules.xml
* .idea/plugin-hub.iml
* .idea/vcs.xml

**Comment by ImToxicCode** [comment]
Did I do this incorrectly ?

**Comment by Nightfirecat** [comment]
Yes, that looks better.

**Comment by Nightfirecat** [comment]
[`client.getVarbitValue()` must be called on the client thread.](https://github.com/ImToxicCode/runekit/blob/3f57448109482d4e48cbbc62ffabaa04e701981e/src/main/java/com/runekit/services/FarmingService.java#L154) This method is run on EDT when config is changed. Please utilize `ClientThread` to address this. (you should run your plugin with `-ea` for assertions to trip during development)

**Pull request merged**